.github/actions/skip/action.yml

```diff
@@ -0,0 +1,76 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+name: "Set Skip Env Var"
+description: "Action to set the SKIP_CI environment variable indicating that we should skip CI jobs"
+inputs:
+  paths:
+    description: >-
+      Set the SKIP_CI environment variable when and only when all the changed files located in one of the path,
+      the paths is shell-style pattern.
+    required: false
+    default: >-
```

Comment:
So md and ui submodule changes wouldn't trigger any CI and e2e tasks anymore, right? Do we need to set more required tasks?

Reply:
> So md and ui submodule changes wouldn't trigger any CI and e2e tasks anymore, right?

Yes, effectively, no CI task will be run when only `*.md` and `skywalking-ui` are updated.

> Do we need to set more required tasks?

We can set all E2E tests as required, WDYT?

Reply:
> We can set all E2E tests as required, WDYT?

I want to, but meter system tests are not stable from time to time. I am not sure why.

Reply:
> > We can set all E2E tests as required, WDYT?
> 
> I want to, but meter system tests are not stable from time to time. I am not sure why.

Well, let's keep it as is for now